plugins/flatpak/gs-flatpak.c

```diff
@@ -3864,6 +3910,13 @@ gs_flatpak_app_install (GsFlatpak *self,
 			return FALSE;
 		}
 
+		/* Check disk space requirements for the list */
+		if (!gs_flatpak_allow_install_or_update (self, app, list, FALSE)) {
```

Comment:
You're not checking if it's an auto update here.

Reply:
This is the `gs_flatpak_app_install`. I check for autoupdating in `gs_flatpak_update_app`. It's what it is supposed to do right?